autogen/agentchat/groupchat.py

```diff
@@ -132,7 +229,7 @@ def manual_select_speaker(self, agents: Optional[List[Agent]] = None) -> Union[A
 
     def _prepare_and_select_agents(
         self, last_speaker: Agent
-    ) -> Tuple[Optional[Agent], List[Agent], Optional[List[Dict]]]:
+    ) -> Tuple[Optional[Agent], List[Agent], Optional[List[Dict]], Optional[List[Dict]]]:
```

Comment:
Is this right? Only three elements are returned.

Reply:
No, this is not right, it should return only three elements indeed.
`-> Tuple[Optional[Agent], List[Agent], Optional[List[Dict]]]`
Thank you.

Reply:
Done